src/instructlab/model/chat.py

```diff
@@ -789,18 +789,35 @@ def chat_cli(
     sys_prompt = CONTEXTS.get(context, "default")(get_model_arch(pathlib.Path(model)))
     loaded["messages"] = [{"role": "system", "content": sys_prompt}]
 
+    retriever: DocumentStoreRetriever | None = None
+
     # Instantiate retriever if RAG is enabled
     if rag_enabled:
         logger.debug("RAG enabled for chat; initializing retriever")
-        retriever: DocumentStoreRetriever | None = create_document_retriever(
-            document_store_uri=document_store_uri,
-            document_store_collection_name=collection_name,
-            top_k=top_k,
-            embedding_model_path=embedding_model_path,
-        )
+        try:
+            retriever: DocumentStoreRetriever | None = create_document_retriever(
+                document_store_uri=document_store_uri,
+                document_store_collection_name=collection_name,
+                top_k=top_k,
+                embedding_model_path=embedding_model_path,
+            )
+        except FileNotFoundError:
+            logger.error(
+                f"Ingested documents were not found at {document_store_uri}.  Try running `ilab rag convert` or setting the value for `--document-store-uri` in chat to point to an existing output of convert."
+            )
+            rag_enabled = False
+        except Exception:  # pylint:disable=broad-exception-caught
+            # We get a generic Exception from the document store if it is pointing to a file that exists but is not a valid vector db file.
```

Comment:
In this case, would it make sense to eliminate the branch you have above where you catch `FileNotFoundError`? It seems like the way you handle this exception is the same in both cases, so I'm not seeing any reason to maintain two identical paths.

Reply:
I wanted a different error message for each of them.  The one where the file is not found is trying to say that there is nothing there while the one for Exception is saying there is something there but it is not a valid DB file.

Reply:
Sorry, I keep reversing myself, but I am tried it out this was and I think it winds up being a sub-optimal experience.   The main flow we want to address here is user tries to run with RAG without first converting and ingesting documents, so we want to provide awareness of that flow in the error in that case.  Your point that the user might not have the embedding model is a good one, but that only fails later in the code anyway so I think it needs separate handling.  I will work on this more and try to come up with a more complete solution.  (Thank you for raising the issue though!)